Profile page unit tests and methods

rrazcueta opened the pull request.

rrazcueta committed:
created login.py
started unitTests_Login.py (this name is ridiculous)
modified usermanagement.html to enforce minlength 4 on username and password

rrazcueta committed:
tweak

rrazcueta committed:
- changed usermanagement.html name requirement to be length 1 again
- adapted testsessionlab's logintests, but realizing login as an object is kind of much.... idk how to do this

rrazcueta committed:
-moved acceptance tests to acceptance tests folder
-added "skills" to models
-created unittests for ProfilePage.TAProfile and otherProfile
-rewrote views.py ProfilePage.post
-wrote ProfilePage.TAProfile and otherProfile

Kinneyj47 commented:
Good way of handling skills. Much more efficient than creating a new model class! Incredible work!

rrazcueta committed:
Merge remote-tracking branch 'origin/master' into profile-page-unit-tests-and-methods

# Conflicts:
#	TAScheduler/views.py

rrazcueta committed:
-moved acceptance tests to acceptance tests folder
-added "skills" to models
-created unittests for ProfilePage.TAProfile and otherProfile
-rewrote views.py ProfilePage.post
-wrote ProfilePage.TAProfile and otherProfile

-fixed AM merge problems

gatesjm-uwm commented:
:+1:, looks a-okay!
